virt/release_notes/virt-4-18-release-notes.adoc

```diff
@@ -150,6 +153,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 * If you clone more than 100 VMs using the `csi-clone` cloning strategy, then the Ceph CSI might not purge the clones. Manually deleting the clones might also fail. (link:https://issues.redhat.com/browse/CNV-23501[*CNV-23501*])
 ** As a workaround, you can restart the `ceph-mgr` to purge the VM clones.
 
+// CNV-55104: 4.18 - Unresolved
+* If you perform storage class migration for a stopped VM, the VM might not be able to start because of a missing bootable device. To prevent this, make sure that you start storage class migration for running VMs. (link:https://issues.redhat.com/browse/CNV-55104[*CNV-55104*])
```

Comment:
```suggestion
* If you perform storage class migration for a stopped VM, the VM might not be able to start because of a missing bootable device. To prevent this, do not attempt storage class migration if the VM is not running. (link:https://issues.redhat.com/browse/CNV-55104[*CNV-55104*])
+
:FeatureName: Storage class migration
include::snippets/technology-preview.adoc[]
```